src/lib/gas.ts

```diff
@@ -3,6 +3,15 @@ import { knexController } from '../database/database';
 import { CreateGasPriceBody, EnrichedGas, Gas } from '../types/gas.types';
 import { convertDateToMariaDBDateTime } from './dateTime';
 
+const ENRICHED_GAS_COLUMNS = [
+  'g.id AS gasId',
+  'g.name AS gasName',
+  'gp.id AS gasPriceId',
+  'gp.price_eur_cents AS priceEurCents',
+  'gp.active_from AS activeFrom',
+  'gp.active_to AS activeTo',
+];
```

Comment:
```suggestion
const ENRICHED_GAS_COLUMNS = [
  'g.id AS gasId',
  'g.name AS gasName',
  'gp.id AS gasPriceId',
  'gp.price_eur_cents AS priceEurCents',
  'gp.active_from AS activeFrom',
  'gp.active_to AS activeTo',
].join(",");
```

Eikös nää voitas joinata jo tässä niin ei tartte sitä joinia kutsua "ajoaikana"

Reply:
Myös toi nimi on toi hassu täälläkin monessa kohtaa